init/config.js

```diff
@@ -47,4 +55,16 @@ module.exports = {
     signupFormUrl: "http://localhost:3000",
     isListed: true,
   },
+  projectWithoutTeam: {
+    slug: "test-dev-recruiting",
```

Comment:
Let's use some different names. For example, you can use 'dev-launchers-platform' project from https://apiv4.devlaunchers.org/api/projects